app/sessionStore.js

```diff
@@ -66,8 +68,7 @@ const getStoragePath = () => {
  * @param {object} payload - Application state as per
  *   https://github.com/brave/browser/wiki/Application-State
  *   (not immutable data)
- * @return a promise which resolves when the state is saved
- */
+ * @return a promise which resolves when the state is saved */
 module.exports.saveAppState = (payload, isShutdown) => {
```

Comment:
same here- can you leave the `*/` on the next line? this matches the style we have in the rest of the app 😄